source/includes/usage-examples/sample-app.rb

```diff
@@ -0,0 +1,18 @@
+require 'bundler/inline'
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = "<connection string URI>"
```

Comment:
I: should be 'connection string' or 'connection URI', but not both. applies to all files